Sources/RInAppMessaging/Extensions/UIColor+IAM.swift

```diff
@@ -1,3 +1,5 @@
+import UIKit
```

Comment:
`import UIKit.UIColor` ?

Reply:
I would have to import `UIStatusBarStyle` as well (for `.lightContent` part).